### PR TITLE
yaml: Lint test yaml

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: yaml: Lint test yaml
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ admin:
   - name: envoy.access_loggers.file
     typed_config:
       "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+      # im wondering if this should just be ""
```
Comment (zuercher) [inline test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml:6]: "/dev/null" maybe? Please add a TODO comment if you're not going to change it.

**Review by zuercher — COMMENTED** [review]
One nit, but looks good otherwise.

File: test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml (hunk 1 above)
Comment (phlax) [reply test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml:6]: apologies for delay on this - im not entirely sure best way to update - ill set to `/dev/null` and if that works lets go with that...

Comment (phlax) [reply test/extensions/filters/network/postgres_proxy/postgres_test_config.yaml]: it turns out that the test is actually expecting `{}` so reverted and removed my comment

**Review by zuercher — APPROVED** [review]
I guess the `{}` is important. Looks good!

**Pull request merged**